src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -1082,7 +1082,7 @@ class PersonAgent(
         log.warning(s"$id is still travelling at end of simulation.")
         log.warning(s"$id events leading up to this point:\n\t${getLog.mkString("\n\t")}")
       } else if (stateName == PerformingActivity) {
-        logger.warn(s"$id is performing Activity at end of simulation")
+        logger.debug(s"$id is performing Activity at end of simulation")
```

Comment:
also keep logger.warn(s"Performing Activity at end of simulation")

Reply:
done